proto/dymension/sequencer/operating_status.proto

```diff
@@ -10,8 +10,12 @@ enum OperatingStatus {
   option (gogoproto.goproto_enum_prefix) = false;
   // OPERATING_STATUS_UNSPECIFIED defines zero-value for status ordering
   OPERATING_STATUS_UNSPECIFIED = 0 [(gogoproto.enumvalue_customname) = "Unspecified"];
-  // OPERATING_STATUS_PROPOSER defines a sequencer that is active and can propose state updates
+  // OPERATING_STATUS_PROPOSER defines a sequencer that is active and expected to send state updates
   OPERATING_STATUS_PROPOSER = 1 [(gogoproto.enumvalue_customname) = "Proposer"]; 
```

Comment:
PROPOSER and BONDED are **not**  mutually exclusive, hence I think its cleaner to move proposer to be a flag (same as jailed)

Reply:
will be handled https://github.com/dymensionxyz/dymension/issues/614